ccb.go

```diff
@@ -0,0 +1,299 @@
+					Birthday *struct {
+						ID string `xml:"id,attr,omitempty" json:"id,omitempty"`
+					} `xml:"birthday,omitempty" json:"birthday,omitempty"`
+					Anniversary *struct {
+						ID string `xml:"id,attr,omitempty" json:"id,omitempty"`
+					} `xml:"anniversary,omitempty" json:"anniversary,omitempty"`
+					Gender *struct {
+						ID string `xml:"id,attr,omitempty" json:"id,omitempty"`
+					} `xml:"gender,omitempty" json:"gender,omitempty"`
+					MaritalStatus *struct {
+						ID string `xml:"id,attr,omitempty" json:"id,omitempty"`
+					} `xml:"marital_status,omitempty" json:"marital_status,omitempty"`
+					UserDefinedFields *struct {
+						ID string `xml:"id,attr,omitempty" json:"id,omitempty"`
+					} `xml:"user_defined_fields,omitempty" json:"user_defined_fields,omitempty"`
+					Allergies *struct {
+						ID string `xml:"id,attr,omitempty" json:"id,omitempty"`
+					} `xml:"allergies,omitempty" json:"allergies,omitempty"`
+				} `xml:"privacy_settings,omitempty" json:"privacy_settings,omitempty"`
+				Active  string `xml:"active,omitempty" json:"active,omitempty"`
+				Creator *struct {
+					ID string `xml:"id,attr,omitempty" json:"id,omitempty"`
+				} `xml:"creator,omitempty" json:"creator,omitempty"`
+				Modifier *struct {
+					ID string `xml:"id,attr,omitempty" json:"id,omitempty"`
+				} `xml:"modifier,omitempty" json:"modifier,omitempty"`
+				Created                   string `xml:"created,omitempty" json:"created,omitempty"`
+				Modified                  string `xml:"modified,omitempty" json:"modified,omitempty"`
+				UserDefinedTextFields     string `xml:"user_defined_text_fields,omitempty" json:"user_defined_text_fields,omitempty"`
+				UserDefinedDateFields     string `xml:"user_defined_date_fields,omitempty" json:"user_defined_date_fields,omitempty"`
+				UserDefinedPulldownFields string `xml:"user_defined_pulldown_fields,omitempty" json:"user_defined_pulldown_fields,omitempty"`
+			} `xml:"individual,omitempty" json:"individual,omitempty"`
+		} `xml:"individuals,omitempty" json:"individuals,omitempty"`
+		FormResponses *struct {
+			Count        string `xml:"count,attr,omitempty" json:"count,omitempty"`
+			FormResponse []*struct {
+				ID   string `xml:"id,attr,omitempty" json:"id,omitempty"`
+				Form *struct {
+					ID string `xml:"id,attr" json:"id,omitempty"`
+				} `xml:"form,omitempty" json:"form,omitempty"`
+				Individual *struct {
+					Name string `xml:",chardata" json:"name,omitempty"`
+					ID   string `xml:"id,attr,omitempty" json:"id,omitempty"`
+				} `xml:"individual,omitempty" json:"individual,omitempty"`
+				Created       string `xml:"created,omitempty" json:"created,omitempty"`
+				Modified      string `xml:"modified,omitempty" json:"modified,omitempty"`
+				ProfileFields *struct {
+					ProfileInfo []*struct {
+						Name string `xml:"name,attr,omitempty" json:"name,omitempty"`
+						Text string `xml:",chardata" json:"text,omitempty"`
```

Comment:
missing xml field name

Reply:
In xml, some fields don't have a name. When it's chardata, it's like the innertext of an HTML element.